.travis.yml

```diff
@@ -3,7 +3,7 @@
 sudo: false
 language: ruby
 cache: bundler
-script: "bundle exec rake validate lint spec"
+script: "bundle exec rake release_checks"
```

Comment:
This change came in when I ran modulesync against this module after my PR simply adding the gemfile was merged. This seems to have been a modulesync run that missed this module in the past. I added it just to keep it up to date.